test/suite/test_rollback_to_stable08.py

```diff
@@ -121,10 +121,7 @@ def test_rollback_to_stable(self):
         self.assertEqual(upd_aborted, 0)
         self.assertEqual(keys_removed, 0)
         self.assertEqual(keys_restored, 0)
-        if self.in_memory:
-            self.assertGreater(pages_visited, 0)
-        else:
-            self.assertEqual(pages_visited, 0)
```

Comment:
This test doesn't expect to be removing any updates, Does our transaction logic is letting to go inside to perform rollback to stable that is not required?

Reply:
Yeah, the older commits were causing this issue. I have reverted this file to the original.